test/functionalTest/harnessFunctions.sh

```diff
@@ -450,7 +450,7 @@ function accumulatorStart()
 
   accumulatorStop $port
 
-  accumulator-server.py $port /notify $bindIp 2> /tmp/accumulator_$port &
+  accumulator-server.py $port /notify $bindIp > /tmp/accumulator_${port}_stdout 2> /tmp/accumulator_$port &
```

Comment:
However, if you use "_stdout" for the standard output after `>`, shouldn't we use "_stderr" for the error output after `2>`?